device_ppc64le.go

```diff
@@ -0,0 +1,11 @@
+//
+// Copyright (c) 2019 Intel Corporation
```

Comment:
ARM Limited?

Reply:
@jcvenegas I'm confused with these copyright header, those no-arm64 code are all copy from the original file with Intel header. so I'm not sure how to write these header. @jodh-intel @grahamwhaley @bergwolf any idea?

Reply:
Thanks for raising this issue @jongwu! I agree that this is confusing, so we need to document this. IANAL so will need to get input from folks more knowledgeable in that area. Since this file is based on an existing one, I would think it is "safe" to always reference the original copyright *as well* (so that you could list both Intel and ARM for 2019).

@grahamwhaley - any thoughts on this? I'm happy to write it up once we have clarity on this subtle area.

As to a location for where to put the words.. I think it would either need to go in:

- https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md

- But, for parity with https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md, it may make sense to create a `https://github.com/kata-containers/documentation/blob/master/Source-File-Requirements.md`

Wherever it goes, it needs to cover the following copyright scenarios:

- Adding a new source file.
- Adding a new source file which is similar to an existing file:
  - Exact copy but for different arch (covering both a tiny fragment (like on this PR) and also a very large and complex file).
  - "Based on" an existing file but with potentially extensive modifications.
- ... ?

Reply:
Thanks @jodh-intel, it is very helpful for me. If I get it right, the copyright header for device_amd64.go, device_ppc64le.go and device_s390x.go should include both intel and ARM. Is that right?
If that is the truth, what's the shape of these header? eg. it should be "Intel Corporation and ARM limited" or "Intel Corporation && ARM limited" or something else?

Reply:
afaik, you would put your new addition copyright on a new line, so like:

```
//
// Copyright (c) 2018 Intel Corporation
// Copyright (c) 2019 <your company/person name here>
//
// SPDX-License-Identifier: Apache-2.0
//
```

and replace `your company` etc. :-)

Reply:
I should probably note - your company may have its own ideas, rules or specific name/format for this btw, and you might want to check internally first.

Reply:
// Copyright (c) 2019 ARM Limited
//
// SPDX-License-Identifier: Apache-2.0
//
plz use this. ;)

Reply:
Thanks @grahamwhaley